charts/rstudio-pm/README.md

```diff
@@ -142,3 +142,6 @@ The Helm `config` values are converted into the `rstudio-pm.gcfg` service config
 | strategy.type | string | `"RollingUpdate"` |  |
 | versionOverride | string | `""` | A Package Manager version to override the "tag" for the RStudio Package Manager image. Necessary until https://github.com/helm/helm/issues/8194 |
 
+----------------------------------------------
+Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
```

Comment:
Ohh, I see now that there's a GHA workflow to update the README. So you shouldn't have to regenerate the README manually?

If you do, it seems like I had a different version of helm-docs installed, so it would be nice to have that version documented somewhere, or in a Docker image perhaps.

Reply:
Yes! You shouldn't need to regenerate manually 😄 Although coming from forks it still fails.

But yes, I just added that to the list of things we need to improve in #72